### ISSUE TITLE
New Download Link? MSPAForums is down!

### ISSUE THREAD

**Issue by delusionalAbberation** [illuminatedwax/pesterchum#186]
Hey there, ghostDunk!
So I recently got a new laptop after my old one crapped out and everything just fried. Today was also the day I found out that MSPAForums is down, and I therefore cannot download Pesterchum. Is there any alternate download link I can use? I'd really like to pester all of my friends and do some roleplaying! Thanks in advance!

**Comment by arcaneAgilmente** [comment]
While the forums are down, feel free to use these links:
Windows: https://app.box.com/s/3hjd3f8r4b9aa9ea6lyn
OSX: https://app.box.com/s/1zu9c4zli5z07w8ah9mx

**Comment by delusionalAbberation** [comment]
Hmmm...It isn't letting me download it. I click the "download" button, but
nothing happens.

**Comment by arcaneAgilmente** [comment]
Hmm, that's odd.
Try this one instead: https://mega.nz/#!poti1IID!V15Q1tUs6P7gLXrV-76WTm_GTIo5IMck9_JD3zaixvY

**Comment by edwanvi** [comment]
If you're crazy enough you could install python and run Pesterchum through the source code in this fine repository.

**Comment by hashbeep** [comment]
@tkdberger How would someone go about doing that?

**Comment by arcaneAgilmente** [comment]
If you have to ask how to compile code, I wouldn’t advise it.

On Thu, Aug 18, 2016 at 4:25 PM -0400, "DavepetaspriteSquared" notifications@github.com wrote:

@tkdberger How would someone go about doing that?

##

**Comment by edwanvi** [comment]
@DavepetaspriteSquared 

Being screwed in the head.

Alternatively, take the following steps.
1. Click the green "Clone or download" button on [this page](https://github.com/illuminatedwax/pesterchum).
2. Choose "Download ZIP". Don't mess with the ZIP file just yet.
3. [Go to Python.org and install Python 3.](https://www.python.org/downloads/release/python-352/) This is the language Pesterchum was written in, and is the easiest way to run from source. **NOTE:** Most Linux distributions come with Python.
4. Ensure Python is working. Open a command prompt (Windows users: Ctrl + R, type `cmd` and hit enter.) Type `python` and hit enter.
5. Get an output like this: 

```
Python 3.2.5 (some date and time) [more info (more info)] on <platform>
Type "help", "copyright", "credits" or "license" from more information.
>>>
```

If you don't have that output, please STOP and ASK ME FOR HELP. If I can't help you, consult Google. You may now close that window.

Pesterchum depends on a library called PyQt5. You can get it pretty easy with `pip3 install PyQt5`. If you get an error message saying that no downloads could be found that satisfy the rquirement then you are probably using an unsupported version of Python.

To run pesterchum:
1. Extract the zip file to wherever you like.
2. Open a command line and navigate to the directory where `pesterchum.py` lives by running `cd path/to/pesterchum.py` or `cd path\to\pesterchum.py` on Windows.
3. Type `python pesterchum.py`
4. ???
5. Profit

**Comment by edwanvi** [comment]
@AratnitY This isn't compiling, this is running.

If you had to compile it, I'd try to compile it and upload binaries.

**Comment by hashbeep** [comment]
Is there a way to make it so I don't have to run that every time I was to use Pesterchum?

**Comment by edwanvi** [comment]
Once you've done that whole thing once you can start from 

> 2. Open a command line and navigate to the directory where pesterchum.py lives by running `cd path/to/pesterchum.py` or `cd path\to\pesterchum.py` on Windows.

whenever you feel like doing a pester.

**Comment by hashbeep** [comment]
Thanks a lot!

**Comment by edwanvi** [comment]
No problem!

**Comment by monabat79** [comment]
You can also download pesterchum from [this page](http://sburb.weebly.com/pesterchum.html). I downloaded it there.

**Comment by wertercatt** [comment]
http://silhavey.com/src/pesterchum3.41.zip Here's the portable ZIP version.

**Comment by arcaneAgilmente** [comment]
@wertercatt I'm going to nitpick here. That's not [portable by definition](http://portableapps.com/about/what_is_a_portable_app) considering it leaves files on the host computer.
All that zip file is is the regular program in an archive that can be extracted to any location on the users computer.

If you want something designed to function off of a USB or other external drive, this is what you want: https://app.box.com/s/a885hxjj4zaxa9h32pi9

**Comment by wertercatt** [comment]
Yeah, I guess I'm just used to using "portable" apps to install things without admin.
